server/api/screenshot/screenshotController.js

```diff
@@ -105,6 +118,13 @@ exports.destroy = function(req, res, next) {
       res.status(404).end();
     } else {
       // If the screenshot exists, remove it
+      // First from Cloudinary, -->TODO: create helper function to remove this from controller
+      // We might want to use { invalidate: true } as seen in cloudinary.com/documentation/node_image_upload
+      // so that the CDN doesn't return a cached copy
+      cloudinary.uploader.destroy(screenshot.originalImageId, function(result) {
+        console.log(result);
```

Comment:
If you want to keep this around, make the log more meaningful: something like `console.log('Deleting screenshot from Cloudinary. Result: ' + result);`